Zend/zend_object_handlers.c

```diff
@@ -275,14 +275,12 @@ static ZEND_COLD zend_never_inline void zend_bad_property_name(void) /* {{{ */
 }
 /* }}} */
 
-static ZEND_COLD zend_never_inline void zend_forbidden_dynamic_property(
-		zend_class_entry *ce, zend_string *member) {
+ZEND_API zend_never_inline void zend_forbidden_dynamic_property(zend_class_entry *ce, zend_string *member) {
 	zend_throw_error(NULL, "Cannot create dynamic property %s::$%s",
 		ZSTR_VAL(ce->name), ZSTR_VAL(member));
 }
 
-static ZEND_COLD zend_never_inline bool zend_deprecated_dynamic_property(
-		zend_object *obj, zend_string *member) {
+ZEND_API zend_never_inline bool zend_deprecated_dynamic_property(zend_object *obj, zend_string *member) {
```

Comment:
Same comment - change this back to `static ZEND_COLD zend_never_inline bool zend_deprecated_dynamic_property(`? Some functionality of zend_deprecated_dynamic_property is specific to the Zend VM's error handling needs